file2.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE users(
+	username       TEXT PRIMARY KEY NOT NULL,
+	email          TEXT    NOT NULL,
```

Comment:
So emails are kind of weird—per the RFCs on them, they are case sensitive. But in practice all email providers ignore this and treat them as case insensitive, so I'd do the same here.